spec/components/validators/integer_setting_validator_spec.rb

```diff
@@ -21,9 +21,21 @@
 
       it "returns true if value is a valid integer" do
         expect(validator.valid_value?(1)).to eq(true)
-        expect(validator.valid_value?(-1)).to eq(true)
         expect(validator.valid_value?('1')).to eq(true)
-        expect(validator.valid_value?('-1')).to eq(true)
+      end
+
+      it "defaults min to 0" do
```

Comment:
I just replaced:

```
expect(validator.valid_value?(100)).to eq(true)
```

with

```
expect(validator.valid_value?(0)).to eq(true)
```

So perhaps that makes it a bit clear? Also, changing min to `5` will result in failing tests now. 😄